backend/app/models/company.rb

```
@@ -211,4 +245,25 @@ def fetch_or_create_stripe_customer_id!
+        name.present?
+      when "add_bank_account"
+        bank_account_ready?
+      when "invite_contractor"
+        company_workers.active.exists?
+      when "send_first_payment"
+        invoices.where(status: Invoice::PAID_OR_PAYING_STATES).exists?
+      when "fill_tax_information"
+        user.user.compliance_info&.tax_information_confirmed_at.present?
+      when "add_payout_information"
+        user.user.bank_account.present?
+            when "sign_contract"
```

RuboCop (1.75.5)

[convention] Indent `when` as deep as `case`.

(Layout/CaseIndentation)